src/wazuh_modules/vulnerability_scanner/testtool/rocksDBQuery/main.cpp

```diff
@@ -21,7 +21,7 @@ int main(const int argc, const char** argv)
     {
         CmdLineArgs cmdLineArgs(argc, argv);
 
-        auto rocksDB = Utils::RocksDBWrapper(cmdLineArgs.getDBPath());
```

Comment:
I've got this error. I don't think is very important in this tool, but do you know the origin?
![image](https://github.com/wazuh/wazuh/assets/13010397/a046139f-70cf-4a46-ac5b-6f0562750b6f)

Reply:
I'll look at this again, thank you.

Reply:
Hi again @MiguelazoDS , I've solved this warning in the last commit.